core/metacling/src/TCling.cxx

```diff
@@ -1790,8 +1790,55 @@ void TCling::LoadPCM(std::string pcmFileNameFullPath)
       Fatal("LoadPCM", "The file %s is not a ROOT as was expected\n", pcmFileName.Data());
       return;
    }
-   TFile pcmFile(pcmFileName + "?filetype=pcm", "READ");
-   LoadPCMImpl(pcmFile);
+   fLoadedRdicts.push_back(new TFile(pcmFileName + "?filetype=pcm", "READ"));
+   TFile* pcmMemFile = fLoadedRdicts.back();
+   LoadPCMImpl(*pcmMemFile);
+}
+
+void TCling::UnLoadPCMImpl(const TFile& rdict)
+{
+   // FIXME: Reverse the effect of LoadPCMImpl.
```

Comment:
@pcanal, I have put all things in place except for this ;)

Reply:
@pcanal, the fate of this PR depends on this.

Reply:
@pcanal, I see you were working in a similar area, maybe you can also look into this and we can land this PR.